Assets/Scripts/Player/PlayerController.cs

```diff
@@ -104,6 +110,8 @@ private void Move(InputAction.CallbackContext obj)
 
                 // Create click point effect
                 Instantiate(clickEffect, hit.point + Vector3.up * 5, Quaternion.identity);
+                
+                playerPositionUpdated.Invoke(transform.position);
```

Comment:
If I understand this correctly, then it will invoke the event after the player clicks on the screen but before the player reaches the new destination, right? So this will give back the position before the player has moved. Is that wanted?

Reply:
Is this even used?

Reply:
it only sends the position at every click for better performance, currently not really used, only for remote logging which is experimental

Reply:
Not really used, which means it can be removed? 👀

Reply:
Nope, I use remote Services to monitor in game events like position change and state change...